gradle/ghPages.gradle

```diff
@@ -1,19 +1,15 @@
-apply plugin: "org.ajoberstar.github-pages"
+apply plugin: 'org.ajoberstar.git-publish'
 
-githubPages {
+gitPublish {
     repoUri = 'https://github.com/johnrengelman/shadow.git'
-    credentials {
-        username = project.hasProperty('githubToken') ? project.githubToken : ''
-        password = ''
-    }
```

Comment:
With the credential configuration being removed, how does they get configured now?

Reply:
Sorry i didn't get to this, but the Migration changes and how to configure it is explained in the link I gave in the original description: https://github.com/ajoberstar/gradle-git-publish